workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -43,6 +45,30 @@ public EntityGameObjectLinker(World world, ILogDispatcher logDispatcher)
             spatialOSComponent.SpatialEntityId = spatialEntityId;
             spatialOSComponent.World = world;
             spatialOSComponent.LogDispatcher = logDispatcher;
+
+            var gameObjectReference = new GameObjectReference { GameObject = gameObject };
+            viewCommandBuffer.AddComponent(entity, gameObjectReference);
+
+            var gameObjectReferenceHandleComponent = new GameObjectReferenceHandle();
+            entityCommandBuffer.AddComponent(entity, gameObjectReferenceHandleComponent);
```

Comment:
QQ: Why do we have `GameObjectReference` and `GameObjectReferenceHandle`? I missed the context on this

Reply:
At this point, there is no need for the `GameObjectReference` component anymore. We just haven't managed to remove it yet but we should eventually. :)

Reply:
GameObjectReferenceHandle is removed in Jess's PR

Reply:
it will be great :D